server/server.js

```diff
@@ -98,19 +99,19 @@ app.post("/api/v1/adoptionApplication", (req, res) => {
     applicationStatus,
     petId
   } = req.body
-  pool
-    .query(
+  pool.query(
       "INSERT INTO adoption_applications(name, phone_number, email, home_status, application_status, pet_id) VALUES($1, $2, $3, $4, $5, $6)",
       [name, phoneNumber, email, homeStatus, applicationStatus, petId]
     )
     .then(result => {
       return res.json(result.rows)
     })
-    .catch(err => {
-      console.log(err)
-      res.sendStatus(500)
+    .catch(error => {
+      console.log(error)
     })
 })
+      
+
```

Comment:
Only need one blank line between functions

Reply:
I'm going to change here ok? for some reason I'm getting the follow error when I try to run by terminal
`To https://github.com/xshigeox/adopt_a_pet.git
 ! [rejected]        head -> login-component-implementation (non-fast-forward)
error: failed to push some refs to 'https://github.com/xshigeox/adopt_a_pet.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.`